components/electric/lib/electric/features.ex

```diff
@@ -90,6 +90,14 @@ defmodule Electric.Features do
     :ok
   end
 
+  def list(name \\ __MODULE__) do
+    name
+    |> :ets.match(:"$1")
+    |> Enum.into(%{}, fn [f] -> f end)
+    |> Map.merge(Process.get({__MODULE__, name}, %{}))
+    |> Enum.to_list()
```

Comment:
If the intent here is to get all table entries, I find `:ets.tab2list()` to be more descriptive. And the temporary conversion to map is not strictly necessary here:

```suggestion
    features = :ets.tab2list(name)
    proc_overrides = Process.get({__MODULE__, name}, %{})
    
    Enum.reduce(proc_overrides, features, fn {key, val}, features ->
      Keyword.put(features, key, val)
    end)
```

Reply:
Or you could do a list conversion which is also super simple

```suggestion
    overrides =
      {__MODULE__, name}
      |> Process.get(%{})
      |> Map.to_list()

    name
    |> :ets.tab2list()
    |> Keyword.merge(overrides)
```

Reply:
Thanks for the reminder re tab2list. I could re-implement this as you say but I personally don't see any great advantage to your alternative versions considering the test-only role of this function.